helper/impls/helper-impl-14/src/main/java/com/linkedin/avroutil1/compatibility/avro14/FieldBuilder14.java

```diff
@@ -35,6 +41,14 @@ public FieldBuilder setDoc(String doc) {
     return this;
   }
 
+  @Override
+  public FieldBuilder setDefault(Object defaultValue) {
+    if (!(defaultValue instanceof JsonNode)) {
+      throw new RuntimeException("Default value in avro 1.4 should be a JsonNode!");
```

Comment:
1 - cover nulls
2 - in the exception also print what this thing _IS_ - not just that it isnt a JsonNode.  also - use FQCNs - both jackson1 and 2 have JsonNode and users will be confused.

Reply:
Right. This logic no longer exists in the newest diff, as we take in real default value object and perform object -> JsonNode conversion now.